src/CatalogApplication/Controllers/RatingsController.cs

```diff
@@ -45,17 +45,25 @@ public async Task<ActionResult> GetRating(string noteId)
         }
 
         /// <summary>
-        /// Creates a new rating
+        /// Creates a new rating for the given note
         /// </summary>
         /// <param name="newRating">The rating object with its associate class attributes</param>
-        /// <returns> Action result for creating a new rating
+        /// <returns> Action Result for the async operation
         /// </returns>
         [HttpPost]
-        public Task<ActionResult> NewRating(Rating newRating)
+        public async Task<ActionResult> NewRating(Rating newRating)
```

Comment:
I would suggest using a verb instead of a noun when creating Http Methods. `NewRating` -> `CreateRating`